.github/workflows/ci.yaml

```diff
@@ -0,0 +1,50 @@
+name: CI
+
+on:
+  # Run the CI on any activity on the main branch as well as on any pull
+  # request activity (e.g when it is created and also when it is updated).
+  pull_request:
+  push:
+    branches: [ "main" ]
+
+jobs:
+  build:
+    name: Build and upload SDK (Linux x86-64)
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout seL4 Core Platform repository
+        uses: actions/checkout@v3
+      - name: Checkout seL4 repository
+        uses: actions/checkout@v3
+        with:
+            repository: BreakawayConsulting/seL4
+            ref: sel4cp-core-support
+            path: seL4
+      - name: Install SDK dependencies
+        run: |
+          sudo apt update
+          sudo apt install \
+            cmake pandoc device-tree-compiler ninja-build \
+            texlive-fonts-recommended texlive-formats-extra libxml2-utils \
+            gcc-aarch64-linux-gnu python3.9 python3-pip python3.9-venv \
+            musl-tools
```

Comment:
From the experience of https://github.com/BreakawayConsulting/sel4cp/issues/13, it might be better to build musl from source instead...